src/notation/internal/notationinteraction.cpp

```diff
@@ -216,6 +216,159 @@ void NotationInteraction::rollback()
     m_undoStack->rollbackChanges();
 }
 
+static std::string MScoreErrorToString(MsError err)
```

Comment:
It would be cool to put this methods in a separate file.
I'm sure it will annoy me. So long to scroll.

Reply:
Done!

Reply:
I meant also the content of checkAndShowMScScoreError